domains/certificates/Query.API/API.UnitTests/MeasurementsSyncer/MeasurementsSyncServiceTest.cs

```diff
@@ -121,4 +124,66 @@ public async Task FetchMeasurements_MeasurementsReceived_MeasurementEventsArePub
         await _fakeMeasurementPublisher.Received().PublishIntegrationEvents(Arg.Any<MeteringPoint>(), Arg.Any<MeteringPointSyncInfo>(),
             Arg.Is<List<Measurement>>(measurements => measurements.Count == 2), Arg.Any<CancellationToken>());
     }
+
+    [Fact]
+    public async Task FetchMeasurements_OldEnoughMeasurement_ForAllMinimumAgesBetweenZeroAndOneHundredSixtyEight_ArePublished()
```

Comment:
Maybe explicit tests of the cases with measurements just around the age of filtering?